### DISCUSSION TITLE
Where is Lethe's Bibliography List?

### DISCUSSION THREAD

**Discussion by MGYBY** [chaos-polymtl/lethe#478] [Q&A] [answered]
Hello all,

Thanks for maintaining such an awesome CFD tool. I would like to learn more about the theoretical background and application of Lethe in publications. However, I cannot find a comprehensive (or even an incomplete) list of bibliography of Lethe. Could anyone inform me of where such a list is? 

Thanks in advance.

BYY

**Comment by shahabgol** [comment]
Hello @MGYBY,

Thank you for your interest in Lethe.

You can find the link (https://www.sciencedirect.com/science/article/pii/S2352711020302922?via%3Dihub) on the main page. Our other publications on the DEM and CFD-DEM solvers will be published soon.

**Reply by MGYBY** [reply]
Hello @shahabgol,

Thanks for your answer. I look forward to more Lethe papers.

**Comment by blaisb** [comment, marked as answer]
Dear @MGYBY , the full publication list is now available in the documentation:
https://lethe-cfd.github.io/lethe/publications.html

**Reply by MGYBY** [reply]
Hello Professor @blaisb,

Thanks for this update. This bibliography list for Lethe really helps.
